CONTRIBUTING.md

```diff
@@ -1,46 +1,145 @@
-# Contributing to bpfd
+# Contributing Guide
 
-Thanks for your help improving the project!
+* [New Contributor Guide](#contributing-guide)
+  * [Ways to Contribute](#ways-to-contribute)
+  * [Find an Issue](#find-an-issue)
+  * [Ask for Help](#ask-for-help)
+  * [Pull Request Lifecycle](#pull-request-lifecycle)
+  * [Development Environment Setup](#development-environment-setup)
+  * [Signoff Your Commits](#signoff-your-commits)
+  * [Pull Request Checklist](#pull-request-checklist)
 
-## Reporting issues
+Welcome! We are glad that you want to contribute to our project! 💖
 
-If you believe you've discovered a bug in bpfd, please check if the bug is
-already known or [create an issue](https://github.com/bpfd-dev/bpfd/issues) on
-github. Please also report an issue if you find documentation that you think is
-confusing or could be improved.
+As you get started, you are in the best position to give us feedback on areas of
+our project that we need help with including:
 
-When creating a new issue, make sure to include as many details as possible to
-help us understand the problem. When reporting a bug, always specify which
-version of bpfd you're using and which version of the linux kernel.
+* Problems found during setting up a new developer environment
+* Gaps in our Quickstart Guide or documentation
+* Bugs in our automation scripts
 
-## Fixing bugs and implementing new features
+If anything doesn't make sense, or doesn't work when you run it, please open a
+bug report and let us know!
 
-Make sure that your work is tracked by an issue or a (draft) pull request, this
-helps us avoid duplicating work.
+## Ways to Contribute
 
-### Commits
+We welcome many different types of contributions including:
+
+* New features
+* Builds, CI/CD
+* Bug fixes
+* Documentation
+* Issue Triage
+* Answering questions on Slack/Mailing List
+* Web design
+* Communications / Social Media / Blog Posts
+* Release management
+
+Not everything happens through a GitHub pull request. Please come to our
+[meetings](TODO) or [contact us](TODO) and let's discuss how we can work
+together.
```

Comment:
```suggestion
Not everything happens through a GitHub pull request. Please come to our
[meetings](https://docs.google.com/document/d/17l96_3NMOQS-1a3gfJPPcTnhTrqJb3KWYxFWbRgD-yk/edit?usp=sharing) or [contact us](https://kubernetes.slack.com/archives/C04UJBW2553) and let's discuss how we can work
together.
```

Reply:
We may want to link to MEETINGS.md here instead of the google doc.